lib/iris/coord_systems.py

```diff
@@ -820,3 +820,51 @@ def as_cartopy_crs(self):
 
     def as_cartopy_projection(self):
         return self.as_cartopy_crs()
+
+
+class Mercator(CoordSystem):
+    """
+    A coordinate system in the Mercator projection.
+
+    """
+
+    grid_mapping_name = "mercator"
+
+    def __init__(self, central_lon=0, ellipsoid=None):
```

Comment:
I copied this from the `Stereographic` class. Other classes use `longitude_of_projection_origin`, which is the same as the CF conventions, so I suggest using that.